### PR TITLE
Add a few more CSS variables

### PR DESCRIPTION
**Pull request by mnot**
see #2

### CODE REVIEW

File: rfc2629.xslt [hunk 1]
```diff
@@ -3835,8 +3836,8 @@
       <xsl:variable name="pos" select="position()" />
       <xsl:if test="$pos &lt; count($lc/myns:item) + 1 or $pos &lt; count($rc/myns:item) + 1">
         <tr>
-          <td class="left"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$lc/myns:item[$pos]/node()" /></xsl:call-template></td>
-          <td class="right"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$rc/myns:item[$pos]/node()" /></xsl:call-template></td>
+          <td class="{css-tleft}"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$lc/myns:item[$pos]/node()" /></xsl:call-template></td>
+          <td class="{$css-tright}"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$rc/myns:item[$pos]/node()" /></xsl:call-template></td>
         </tr>
```
Comment (reschke) [inline rfc2629.xslt]: This part appears to be bogus (the variables are not declared).

Comment (mnot) [inline rfc2629.xslt]: they're declared above, by you... (fixed missing $)

File: rfc2629.xslt [hunk 2]
```diff
@@ -3835,8 +3838,8 @@
       <xsl:variable name="pos" select="position()" />
       <xsl:if test="$pos &lt; count($lc/myns:item) + 1 or $pos &lt; count($rc/myns:item) + 1">
         <tr>
-          <td class="left"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$lc/myns:item[$pos]/node()" /></xsl:call-template></td>
-          <td class="right"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$rc/myns:item[$pos]/node()" /></xsl:call-template></td>
+          <td class="{$css-tleft}"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$lc/myns:item[$pos]/node()" /></xsl:call-template></td>
+          <td class="{$css-tright}"><xsl:call-template name="copynodes"><xsl:with-param name="nodes" select="$rc/myns:item[$pos]/node()" /></xsl:call-template></td>
         </tr>
       </xsl:if>
```
Comment (reschke) [inline rfc2629.xslt]: but "left" != "tleft" in the CSS...

Comment (mnot) [inline rfc2629.xslt]: Ah. It seemed like it would be one. Hm.

Comment (reschke) [inline rfc2629.xslt]: I wouldn't rule out that the CSS can be simplified though.

**Comment by mnot** [comment]
OK. Tell me if you want me to try to do something about that; otherwise I think it's good for now (specs on httpwg.github.io now use this + rfcbootstrap). 

Probably going to sleep soon (heavy jet lag).

**Pull request closed without merging**